eng/pipelines/docindex.yml

```diff
@@ -6,7 +6,6 @@ jobs:
   - job: UpdateDocsMsBuildConfig
     pool:
       name: azsdk-pool-mms-ubuntu-2004-general
-      vmImage: MMSUbuntu20.04Compliant
```

Comment:
Is there a reason you removed this? I realize it might not be used but I'm curious if there is a reason you removed it.

Reply:
The image used in pool might use some other image which might not the same as the one we stated here. 
I once tested pool-2022, the image I put in yaml is Win-Compliant, but the one really in use is Win-TLS in pipeline log. The mismatch there made me think it might not need vmImage here in yaml.

Reply:
vmImage is currently only used by the standard DevOps hosted pool. When using the 1ES pools it serves the purpose of documentation only and in some pipelines we use it for the verify-agent-os scripts. In either case I don't really like us having "MMSUbuntu20.04Compliant" in our yml files. I would prefer it simply be "Ubuntu20.04" to help document the OS we are running on.